src/sentry/models/sentryapp.py

```diff
@@ -51,7 +51,10 @@ class SentryApp(ParanoidModel, HasApiScopes):
     uuid = models.CharField(max_length=64,
                             default=default_uuid)
 
-    webhook_url = models.TextField()
+    redirect_url = models.URLField()
```

Comment:
Let's make this one `null=True` for anyone who's service can function completely without User interaction. The `webhook_url` would be enough for that.